src/sentry/api/serializers/models/group.py

```diff
@@ -148,10 +148,37 @@ def get_attrs(self, item_list, user):
             environment = self.environment_func()
         except Environment.DoesNotExist:
             user_counts = {}
+            first_seen = {}
+            last_seen = {}
+            times_seen = {}
```

Comment:
This is a little clunky but I couldn't really think of a nicer way to handle both of these cases (get the data from `Group` vs `GroupTagValue`.)